libs/vm-modules/src/ml/model/model.cpp

```diff
@@ -161,19 +157,17 @@ void VMModel::CompileSequential(fetch::vm::Ptr<fetch::vm::String> const &loss,
 void VMModel::CompileSimple(fetch::vm::Ptr<fetch::vm::String> const &        optimiser,
                             fetch::vm::Ptr<vm::Array<math::SizeType>> const &in_layers)
 {
-  OptimiserType const optimiser_type = ParseName(optimiser->string(), optimisers_, "optimiser");
-  if (optimiser_type != OptimiserType::ADAM)
+  size_t const total_hidden_layers = in_layers->elements.size();
```

Comment:
std::size_t

Reply:
also calling elements.size() is very cheap - you don't need to store it to a variable. but it's fine if you think it's clearer to read that way

Reply:
Yeap, it's more about naming than cacheing :)